tools/tokenserver/purge_old_records.py

```diff
@@ -87,35 +86,24 @@ def purge_old_records(
             range_msg = ""
             if uid_range:
                 range_msg = (
-                    f" within range {uid_range[0] or 'Start'}"
-                    f" to {uid_range[1] or 'End'}"
+                    f" within range {uid_range[0] or 'Start'}" f" to {uid_range[1] or 'End'}"
```

Comment:
Ok, odd format.

Let's just join these two `f""` strings and maybe we can drop the parens?